flake.nix

```diff
@@ -90,6 +90,24 @@
               done
             '';
           };
+
+          build-dist = pkgs.writeShellApplication {
+            name = "build-dist";
+            runtimeInputs = with pkgs; [ nodejs gettext git ];
+            text =
+              let
+                python-dev = self.devShells.${system}.evap-dev.passthru.venv;
+                python-build = self.packages.${system}.python3.withPackages (ps: [ ps.build ]);
+              in
+              ''
+                set -x
+                npm ci
+                ${python-dev}/bin/python ./manage.py compilemessages
+                ${python-dev}/bin/python ./manage.py scss --production
+                ${python-dev}/bin/python ./manage.py ts compile --fresh
+                ${python-build}/bin/python -m build
```

Comment:
We now use nix, npm, uv, hatch/hatchling, and build. Granted, nix is just for dev environments. But still, do we _need_ them all? uv promises that it can create distributions? If there's some specific thing we need missing it would be nice to have this documented. It would be nice in general to have a very brief explanation of the job of every tool used here.

Reply:
I believe uv also has some ideas on how to build and publish python projects, but I wanted to keep that separate and follow the general guidelines from the python package authority: https://packaging.python.org/en/latest/guides/distributing-packages-using-setuptools/#packaging-your-project

This allows us to remove uv if we don't want it anymore for locking dependencies. Hatch (and even more so `build`) seems like the blessed option right now and in general rather stable. So while we don't need it, I think it is the better option right now